app/controllers/facilities_controller.rb

```diff
@@ -1,46 +1,51 @@
 class FacilitiesController < ApplicationController
-    def index
-        @facilities = Facility.all
-    end
+  before_action :authenticate_user!, only: [:edit, :update, :destroy, :base_params, :create_params, :update_params]
+  def index
+    @facilities = Facility.all
+  end
     
-    def new
-        @facility = Facility.new
-    end
+  def new
+    @facility = Facility.new
+  end
     
-    def create
-        facility = Facility.new
-        facility.name = params[:facility][:name]
-        facility.city = params[:facility][:city]
-        facility.state = params[:facility][:state]
-        facility.zip = params[:facility][:zip]
-        facility.specialty = params[:facility][:specialty]
-        facility.save
-        redirect_to "/"
-        
-    end
+  def create
+    @facility = Facility.create(create_params)
+    @facility.save
+  redirect_to "/"
```

Comment:
indent this and instead of using the string "/" use the helper method `root_path`

Reply:
Changed / to helper methods in admins, items, facilities and procedures controllers.